grails-forge-core/src/main/java/org/grails/forge/feature/view/Scaffolding.java

```diff
@@ -57,12 +57,12 @@ public String getDescription() {
 
     @Override
     public String getDocumentation() {
-        return "https://grails.github.io/scaffolding/latest/groovydoc/";
+        return "https://github.com/grails/scaffolding";
```

Comment:
Can we do this instead: `return "https://docs.grails.org/" + VersionInfo.getGrailsVersion() + "/guide/single.html#scaffolding";`